### ISSUE TITLE
(http://b/70031363): Accurate estimator for Softmax needed 

### ISSUE THREAD

**Issue by piascikj** [imdone/tensorflow#2006]
Change to EXPECT_FALSE when the above TODOs are done:

----
*opened via [imdone.io](https://imdone.io) from a code comment on [d0ae1064ed0bb4bd1aed00afd4235f4dd5c853f0](https://github.com/imdone/tensorflow/commit/d0ae1064ed0bb4bd1aed00afd4235f4dd5c853f0) by Max Galkin maxgalkin@google.com*

----

https://github.com/imdone/tensorflow/blob/8cb6e535f6ea14b380aa86d425169adad682cb8c/tensorflow/core/grappler/costs/analytical_cost_estimator_test.cc#L109-L116
